examples/apps/collaborative-textarea/jest.config.cjs

```diff
@@ -3,34 +3,7 @@
  * Licensed under the MIT License.
  */
 
-// Get the test port from the global map and set it in env for this test
-const testTools = require("@fluidframework/test-tools");
+const getBaseJestConfig = require("@fluid-private/test-tools/getBaseJestConfig.cjs");
```

Comment:
Let's use the normal module pattern with `exports` from `test-tools` package and move beyond Node10.
Especially since there are now two exports. Is the default one is still used in places, I guess.

Reply:
The old default export wasn't necessary anymore, the only thing other packages need to access from test-tools is the `getBaseJestConfig()` function, so left only that in the package exports. I did have to reach directly into the package's `dist/` folder from `getBaseJestConfig.cjs` though, which I didn't love but not sure if it can be worked around? Trying to reach to the corresponding `.ts` file result in complaints about seeing `imports` in a CJS context.